packages/components/src/radio-control/README.md

```diff
@@ -1,11 +1,59 @@
 # RadioControl
 
-RadioControl component is used to generate radio input fields.
+Use radio buttons when you want users to select one option from a set, and you want to show them all the available options at once.
 
+![Selected and unselected radio buttons](./radio.png)
+Selected and unselected radio buttons
+
+## Table of contents
+
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+3. [Related components](#related-components)
+
+# Design guidelines
+
+## Usage
+
+### When to use radio buttons
+
+Use radio buttons when you want users to:
+
+- Select a single option from a list.
+- Expose all available options.
+
+If you have a list of available options can be collapsed, consider using a dropdown menu instead — it uses less space.
+
+![Use radio buttons when only one item can be selected from a list.](./radio-usage-do.png)
+**Do**
+Use radio buttons when only one item can be selected from a list.
+
+![Don’t use checkboxes when only one item can be selected from a list. Use radio buttons instead.](./radio-usage-dont.png)
+**Don’t**
+Don’t use checkboxes when only one item can be selected from a list. Use radio buttons instead.
+
+### Defaults
+
+When using radio buttons **one must be selected by default** (i.e., when the page loads, in the case of a web application).
```

Comment:
I'm not sure this is true. I know it's recommended, but we don't enforce it in the component and it is possible that a user could program a "reset" button which allows a radio list to be in a "null" state without a selection.

If we're just recommending this as a "best practices", "must" could be replaced with "should".